proposal.rst

```diff
@@ -237,7 +264,11 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+April 24, 2018 - May 14, 2018	familiarilize myself with xapian by doing some small changes or demoes
+May 15, 2018 - June 6, 2018		improve DBN with EM and test
+June 7, 2018 - June 30, 2018	implement DCM and IAM and test
+July 1, 2018n - July 21, 2018	finalize letor API with omega and test
+July 21, 2018n - August 6, 2018	explore more recent proposed ideas to improve
```

Comment:
This is much too coarse.  As the comment here says:

> The parts should be AT MOST a week long, or else you won't be able to realistically judge how long they might take.  Even a week is too long really.  Try to break larger tasks down into sub-tasks.

But the real problem is the lack of analysis of the problem in the section above.  Without a concrete plan for how you're going to solve the problem, you can't come up with any sort of realistic timescales for doing it.  If you don't have any sort of realistic timescales for it, then how can either you or we have any well-founded confidence that you can pull this off in the 3 months available?

Reply:
I apologized for the hurried version. I made the plan more detailed in new version but I have missed the deadline on website to update it. Thus, I wanna submit it here. https://github.com/changlinzhang/xapian-gsoc-proposal/blob/master/proposal.rst